### DISCUSSION TITLE
What versions of IOS Crashlitics SDK is being used by the RN Library?

### DISCUSSION THREAD

**Discussion by gabriel-tentaculo** [invertase/react-native-firebase#4790] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/70529565/104934296-bad6dd80-5988-11eb-82bf-ca8d965038f2.png)

I have seen this message on my crash logs on the developer portal ARN messages, but as you can see, I cannot view the crash log I was looking for as the Firebase IOS SDK is lower than 4.0 as the message says.

Where can I find which versions of the RN library correspond to the Firebase IOS SDK.?

BTW I'm using ("@react-native-firebase/crashlytics": "8.4.2")

I case such a list does not exist, what then would be the minimum RN Library version that I would have to use to make sure it then comes with the IOS Firebase SDK 4.0+ ?

**Comment by mikehardy** [comment, marked as answer]
If you want support for react-native-firebase issues, use the current stable version - that's currently 10.4.x
Don't look at / focus on "what's the least I can get away with", that's an inefficient strategy. Upgrade to current stable.

**Reply by gabriel-tentaculo** [reply]
Thanks for the reply @mikehardy
